frontend/src/main/components/Nav/AppNavbar.js

```diff
@@ -90,14 +90,14 @@ export default function AppNavbar({ currentUser, systemInfo, doLogout, currentUr
                 )
               }
               {
-                !hasRole(currentUser, "ROLE_DRIVER") && isParticipant(currentUser) && (
+                !hasRole(currentUser, "ROLE_DRIVER") && isParticipant(currentUser) && !hasRole(currentUser, "ROLE_RIDER") && (
```

Comment:
Wouldn't it be easier to say: 

```suggestion
                hasRole(currentUser, "ROLE_ADMIN") (
```

That would be a lot more clear.  Is there a case where that's the wrong logic?   What you have here seems needlessly complex, and it would also prevent Admins that are also drivers or riders from accessing the menu which I don't think we want to do.